pkg/graph/schema.resolvers.go

```diff
@@ -1506,7 +1506,12 @@ func (r *queryResolver) CedarSystemBookmarks(ctx context.Context) ([]*models.Ced
 	return cedarSystemBookmarks, nil
 }
 
-func (r *queryResolver) Deployments(ctx context.Context, systemID string, deploymentType *string, state *string, status *string) ([]*models.CedarDeployment, error) {
+func (r *queryResolver) Deployments(ctx context.Context, cedarSystemID string, deploymentType *string, state *string, status *string) ([]*models.CedarDeployment, error) {
+	cedarSystem, err := r.cedarCoreClient.GetSystem(ctx, cedarSystemID)
```

Comment:
As we discussed in the meeting, calling `GetSystem()` to map `cedarSystemID` to a system's version ID should go in the CEDAR Core client.